package-scripts.yml

```diff
@@ -0,0 +1,48 @@
+scripts:
```

Comment:
Hmm... Maybe I'm confused... But that's not what I meant. I'm just not sure why you're adding a package-scripts.yml file to this repo.